packages/swapkit/chainflip/package.json

```
@@ -0,0 +1,60 @@
+{
+  "author": "swapkit",
+  "dependencies": {
+    "@chainflip/sdk": "1.1.5",
+    "@polkadot/api": "10.11.2",
+    "@polkadot/keyring": "^12.6.2",
+    "@polkadot/util": "^12.6.2"
+  },
+  "description": "SwapKit Lib chainflip",
+  "devDependencies": {
+    "@internal/config": "workspace:*",
+    "@polkadot/types": "10.11.2",
+    "@swapkit/helpers": "workspace:*",
+    "@swapkit/toolbox-evm": "workspace:*",
+    "@swapkit/toolbox-substrate": "workspace:*",
+    "@vitest/coverage-istanbul": "1.1.0",
+    "vite": "5.0.12",
+    "vitest": "1.1.0"
+  },
+  "eslintConfig": {
+    "extends": "../../../internal/eslint-config"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/index.es.js",
+      "require": "./dist/index.cjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "files": [
+    "src/",
+    "dist/"
+  ],
+  "homepage": "https://github.com/thorswap/SwapKit",
+  "license": "Apache-2.0",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.es.js",
+  "name": "@swapkit/chainflip",
+  "peerDependencies": {
+    "@polkadot/types": "10.11.2",
```

In peer we need to accept any higher version as it produces errors and locks developers from updating their own deps 😅

```suggestion
    "@polkadot/types": "^10.11.2",
```